scriptharness/__init__.py

```diff
@@ -185,11 +193,11 @@ def get_actions(all_actions):
     """Build a tuple of Action objects for the script.
 
     Args:
-      all_actions (data structure): ordered mapping of action_name:enabled
+      all_actions (Sequence[str, bool]): ordered mapping of action_name:enabled
```

Comment:
From what I read in this case it would either be a Sequence or an Iterable, but to be honest the difference is also quite blurry for me. Do you think it would be better to have it as Iterable?